integration_tests/__init__.py

```diff
@@ -55,6 +56,16 @@ def tearDown(self):
         assert self.container_id
         execute('docker', 'stop', self.container_id)
         execute('docker', 'rm', self.container_id)
+        self._reset_logging()
+
+    @staticmethod
+    def _reset_logging():
```

Comment:
@mwhamgenomics Just out of interest, when would this be used?

Reply:
The integrations tests test each module by running their `main` method, which most of the time sets up logging handlers. When multiple main methods (and therefore logging setups) are done within one Python session, the logging handlers are duplicated, meaning that messages start getting logged multiple times.

`_reset_logging` should reduce the amount of spam from failing integration tests, and could be moved to EGCG-Core `app_logging` in the future.